src/IIIFPresentation/Repository/Paths/IPresentationPathGenerator.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Repository.Paths;
+
+public interface IPresentationPathGenerator
+{
+    /// <summary>
+    /// Generate full path for IIIF Presentation resources
+    /// </summary>
+    public string GetPresentationPathForRequest(string presentationServiceType, int? customerId = null,
```

Comment:
Should there be 2 overloads here? This takes 2 params: `string? hierarchyPath = null, string? resourceId = null` but every call will only provide `hierarchyPath` OR `resourceId`.